app/models/user.rb

```diff
@@ -242,6 +242,9 @@ def student? ; false end
   def employer? ; false end
   def operator? ; false end
   def school_management?; false end
+  def already_signed?(internship_aggreement_id:); false end
```

Comment:
has_already_signed? est plus clair. Là on dirait que c'est l'utilisateur qui est signé (is_already_signed?)

Reply:
alors c'est une convention ruby avec laquelle je me bats aussi car je suis tenté de suivre les conventions javascript qui intitulent les variables comme tu dis. Avec ruby on a le point d'interrogation qui permet d'éviter ça (le is et le has)... Des conventions de langages qu'on a suivies jusqu'ici, et qu'on va poursuivre je pense